server/src/main/java/org/jboss/as/server/ServerEnvironment.java

```diff
@@ -507,14 +543,14 @@ else if (!tmp.mkdirs()) {
         WildFlySecurityManager.setPropertyPrivileged(HOST_NAME, hostName);
         WildFlySecurityManager.setPropertyPrivileged(SERVER_NAME, serverName);
         WildFlySecurityManager.setPropertyPrivileged(NODE_NAME, nodeName);
-        WildFlySecurityManager.setPropertyPrivileged(HOME_DIR, homeDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(MODULES_DIR, modulesDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_BASE_DIR, serverBaseDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_CONFIG_DIR, serverConfigurationDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_DATA_DIR, serverDataDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_DEPLOY_DIR, serverContentDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_LOG_DIR, serverLogDir.getAbsolutePath());
-        WildFlySecurityManager.setPropertyPrivileged(SERVER_TEMP_DIR, serverTempDir.getAbsolutePath());
+        setPathProperty(HOME_DIR, homeDir);
+        setPathProperty(MODULES_DIR, modulesDir);
+        setPathProperty(SERVER_BASE_DIR, serverBaseDir);
+        setPathProperty(SERVER_CONFIG_DIR, serverConfigurationDir);
+        setPathProperty(SERVER_DATA_DIR, serverDataDir);
+        setPathProperty(SERVER_DEPLOY_DIR, serverContentDir);
+        setPathProperty(SERVER_LOG_DIR, serverLogDir);
+        setPathProperty(SERVER_TEMP_DIR, serverTempDir);
 
```

Comment:
The [documentation](https://docs.jboss.org/author/display/WFLY8/General+configuration+concepts#Generalconfigurationconcepts-Paths) indicates these should be set automatically. We should either set them or we'll need to update the documentation.